go/chat/server.go

```diff
@@ -2422,12 +2423,19 @@ func (h *Server) sendRemoteNotificationSuccessful(ctx context.Context, pushIDs [
 		}
 		conn = rpc.NewTLSConnection(rpc.NewFixedRemote(uri.HostPort),
 			[]byte(rawCA), libkb.NewContextifiedErrorUnwrapper(h.G().ExternalG()),
-			&remoteNotificationSuccessHandler{}, libkb.NewRPCLogFactory(h.G().ExternalG()), h.G().Log,
-			rpc.ConnectionOpts{})
+			&remoteNotificationSuccessHandler{}, libkb.NewRPCLogFactory(h.G().ExternalG()),
+			logger.LogOutputWithDepthAdder{
+				Logger: logger.LogOutputWithDebugAndInfoGuards{
+					Logger: h.G().Log,
+				}}, rpc.ConnectionOpts{})
```

Comment:
Please wrap this stack into a function:
```go
func makeGuardedRPCLogger(l logger.Logger) logger.LogOutputWithDepthAdder {
    return logger.LogOutputWithDepthAdder{
        Logger: logger.LogOutputWithDebugAndInfoGuards{
            Logger: l,
        },
    }
}

Reply:
This is used in multiple packages, so the function will be in `logger` I guess. Will do.